packages/material-ui/src/utils/useIsFocusVisible.js

```diff
@@ -133,7 +132,7 @@ function handleBlurVisible() {
 
 export default function useIsFocusVisible() {
   const ref = React.useCallback((instance) => {
```

Comment:
Do we need to rename the argument? Maybe we could start from the final name.

```suggestion
  const ref = React.useCallback((node) => {
```

Reply:
done